PicturesVIP/PicturesVIP/PictureModule/PictureInteractor.swift

```diff
@@ -0,0 +1,50 @@
+// PictureInteractor.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Интерактор модуля главного экрана
+final class PictureInteractor: PictureBisinessLogic, PictureDataStore {
+    // MARK: - Public Properties
+    
+    var text: String?
+    var presenter: PicturePresentationLogic?
+    var networkWorker: PictureWorkerLogic?
+    
+    // MARK: - Initializers
+    
+    init(_ presenter: PicturePresentationLogic,
+         _ networkWorker: PictureWorkerLogic) {
+        self.presenter = presenter
+        self.networkWorker = networkWorker
+    }
+    
+    // MARK: - Public Methods
+    
+    func requestInitForm(_ request: PictureModel.InitForm.Request) {
+        networkWorker?.getData() { [weak self] result in
+            guard let self else { return }
+            switch result {
+            case let .success(result):
+                self.text = result.explanation
+                self.fetchImageView(result.url)
+                let response = PictureModel.InitForm.Response(picture: result)
+                self.presenter?.presentPicture(response)
+            case let .failure(error):
+                print(error.localizedDescription)
```

Comment:
```suggestion
                print(error.localizedDescription)
```
Принт нужно убрать
Может получится в клоужер передать ошибку ?